worker/src/RTC/Producer.cpp

```diff
@@ -605,6 +607,56 @@ namespace RTC
 		}
 	}
 
+	void Producer::HandleNotification(PayloadChannel::Notification* notification)
+	{
+		MS_TRACE();
+
+		switch (notification->eventId)
+		{
+			case PayloadChannel::Notification::EventId::PRODUCER_SEND:
+			{
+				const auto* data = notification->payload;
+				auto len         = notification->payloadLen;
+
+				// Increase receive transmission.
+				this->listener->OnProducerReceiveData(this, len);
+
+				if (len > RTC::MtuSize + 100)
+				{
+					MS_WARN_TAG(rtp, "given RTP packet exceeds maximum size [len:%zu]", len);
+
+					return;
```

Comment:
Please use `break` instead of `return` within `case` everywhere (there are more like this).

Reply:
It was like that before (this code was moved), but changed as suggested